internal/config/scanner/scanner.go

```diff
@@ -69,6 +84,26 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 	if err = config.CheckValidKeys(config.ScannerSubSys, kvs, DefaultKVS); err != nil {
 		return cfg, err
 	}
+
+	// Stick to loading deprecated config/env if they are already set
+	if kvs.Get(Delay) != "" && kvs.Get(MaxWait) != "" && kvs.Get(Cycle) != "" {
+		return lookupDeprecatedScannerConfig(kvs)
+	}
+
+	switch speed := env.Get(EnvSpeed, kvs.GetWithDefault(Speed, DefaultKVS)); speed {
+	case "fast":
+		cfg.Delay, cfg.MaxWait, cfg.Cycle = 0, 0, 0
+	case "auto":
```

Comment:
```suggestion
	case "default":
```

It isn't really any more auto than the others.